Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeViews.mm

```diff
@@ -392,6 +396,46 @@ + (Class)layerClass
 
 #endif
 
+#if HAVE(MATERIAL_HOSTING)
+
+@implementation WKMaterialHostingView {
```

Comment:
why can't this be in Swift again?

Reply:
So that it can subclass `WKCompositingView` and conform to `WKContentControlled`.